doc/proxy-clients.rst

```diff
@@ -158,6 +159,44 @@ default ``xkey-softpurge``.
     the ``ResponseTagger``, set it up with a
     :ref:`custom TagHeaderFormatter <response_tagger_optional_parameters>`.
 
+Fastly Client
+~~~~~~~~~~~~~~
+
+The Fastly client sends HTTP requests with the ``HttpDispatcher``. Create the
+dispatcher as explained above and pass it to the Fastly client::
+
+    use FOS\HttpCache\ProxyClient\Fastly;
+
+    $varnish = new Fastly($httpDispatcher);
+
+.. note::
+
+    Unlike other supported proxies there is no configuration needed for the proxy itself as all invalidation is done
+    against `Fastly Purge API`_. But for optimal use make sure to tune configuration together with Fastly.
+
+You need to pass the following options to the Fastly client:
+
+* ``service_identifier``: Identifier for your Fastly service account.
+* ``authentication_token``: User token for authentication against Fastly APIs.
+* NB: To be able to clear all cache(`->clear()`), you'll need a token for user with Fastly "Engineer permissions".
```

Comment:
very cool! this is exactly what such documentation should help people with, remove wtf moments :+1: